.gitignore

```diff
@@ -7,5 +7,6 @@
 /build
 /captures
 .externalNativeBuild
+/.idea/misc.xml
```

Comment:
This is a duplicate. You can remove this line

Reply:
Ok, I have removed it.